fee_allocator/fee_allocator.py

```diff
@@ -599,6 +609,99 @@ def generate_bribe_payload(
         builder.output_payload(output_path)
         
         return output_path
+    
+    def _process_hiddenhand_bribes(self, bribe_df, total_bribe_usdc, usdc, bal_bribe_market, aura_bribe_market):
+        usdc.approve(self.book["hidden_hand2/bribe_vault"], total_bribe_usdc + 1)  # 1 wei buffer
+        
+        for _, row in bribe_df.iterrows():
+            if int(row["amount"]) == 0:
+                continue
+                
+            prop_hash = self._get_prop_hash(row["platform"], row["target"])
+            mantissa = round(row["amount"] * 1e6)
+            
+            if row["platform"] == "balancer":
+                bal_bribe_market.depositBribe(prop_hash, self.book["tokens/USDC"], mantissa, 0, 2)
+            elif row["platform"] == "aura":
+                aura_bribe_market.depositBribe(prop_hash, self.book["tokens/USDC"], mantissa, 0, 1)
+    
+    def _process_paladin_quests(self, bribe_df, total_bribe_usdc, usdc):
+        PALADIN_QUEST_BOARDS = {
+            "balancer": "0x8b2ba835056965808aD88e7Ad7866BD57aE75839",
+            "aura": "0x653D8f14292A1C5239d6183b333De1F2e8669310"
+        }
+        
+        bal_bribes = bribe_df[bribe_df["platform"] == "balancer"]
+        aura_bribes = bribe_df[bribe_df["platform"] == "aura"]
+        
+        quest_boards = {}
+        platform_fee_ratios = {}
+        
+        if not bal_bribes.empty:
+            quest_boards["balancer"] = SafeContract(
+                PALADIN_QUEST_BOARDS["balancer"],
+                abi_file_path=f"{base_dir}/abi/paladin_quest_board.json"
```

Comment:
if you use `abi=` here instead of `abi_file_path=` then you only have to load the json once